modules/ROOT/pages/database-administration/aliases/remote-database-alias-configuration.adoc

```diff
@@ -4,65 +4,70 @@
+They assume that you have already installed and started both DBMSs: a local *DBMS A* and a remote *DBMS B*, and that you have administrator access to both systems.
 
-== Setup example
+By creating a remote alias, you define:
 
-In this example, _Alice_ is an administrator and _Carol_ is a user who needs access to a database managed by _Bob_:
+* Which user of the remote *DBMS B* is used.
+* Where the remote database is located.
+* How to connect to the remote database using driver settings.
 
-image::remote-alias-overview.svg[title="Overview of the required remote database alias setup", role="middle"]
+When creating the remote database alias, it can be configured to authenticate with either:
 
-A remote alias defines:
+* `STORED NATIVE CREDENTIALS`, the credentials of a single native user on the remote *DBMS B*.
+* label:new[Introduced in 2025.x] `OIDC CREDENTIAL FORWARDING`, forwarding the bearer authentication token from the logged-in user on the local *DBMS A*.
+The user needs to be logged in with an identity provider supporting OIDC.
 
-* Which user of the remote **DBMS B** is used.
-* Where the remote database is located.
-* How to connect to the remote database using driver settings.
 
-[NOTE]
-====
+[[setup-example-stored-native-credentials]]
+== Setup example with stored native credentials
+
+In this example, _Alice_ is an administrator and _Carol_ is a user who needs access to a database managed by _Bob_.
+
+image::remote-alias-overview.svg[title="Overview of the required remote database alias setup when using stored credentials", role="middle"]
+
 A remote database alias is only accessible to users with appropriate privileges.
-In the example above, _Bob_ is the administrator responsible for deciding which databases (`Db1` or `Db2`) the remote aliases can write and/or read.
+In this example, _Bob_ is the administrator responsible for deciding which database (`db1` or `db2`) the remote aliases can write and/or read.
 Meanwhile, _Alice_ is the administrator that assigns who has access to the privileges set by _Bob_.
 In the example, _Alice_ will assign that access to _Carol_.
+See xref:authentication-authorization/dbms-administration.adoc[DBMS privileges] for more information.
 
-See lxref:authentication-authorization/dbms-administration.adoc[DBMS privileges] for more information.
-====
 
-_Carol_ can use her own regular credentials to access the remote database `Db1` in DBMS after _Alice_ assigns this privilege to her user profile.
-This configuration will also allow _Carol_ to access `Db2` in **DBMS B**.
-If the administrators decide this should not be the case, then _Bob_ must define the appropriate privileges (see xref:authentication-authorization/index.adoc[Authentication and authorization] for further information).
+_Carol_ can use her own regular credentials to access the remote database `db1` in *DBMS B* after _Alice_ assigns this privilege to her user profile.
```

Comment:
If Carol can access db1 directly, why would she need the access to the remote alias of this db1?